README.md

```diff
@@ -2,8 +2,12 @@
 
 Monorepo for the sites in the Svelte ecosystem
 
-## Running
+## Setup
 
 ```
 git submodule update --init --recursive
 ```
+
+You can now edit the content within each directory in [sites](sites) and see it live on your local machine.
+
+Commit as normal and submodule content will be added to the corresponding Svelte project repositories.
```

Comment:
maybe add something about pushing and opening a PR?

Reply:
i'm guessing you need to create a branch first? I saw something about detached head in the chat which I wouldn't know how to deal with

Reply:
i think the branch stuff probably goes without saying. in any case this is all temporary